.github/blunderbuss.yml

```diff
@@ -1,6 +1,8 @@
 assign_issues:
   - sofisl
   - bcoe 
```

Comment:
@sofisl depending on @steffnay's direction, it might make sense to remove you and I, and add @stephenplusplus?

@steffnay who would you like to help triage on this repo?

Reply:
Yeah, @stephenplusplus would be really helpful.